pkg/extension/workspace/fs/interfaces.go

```diff
@@ -20,8 +20,8 @@ const (
 	extensionMetaDir = ".getenvoy/extension"
 )
 
-// WorkspaceDir represents a directory with an extension created by getenvoy toolkit.
-type WorkspaceDir interface {
+// ExtensionDir represents a directory with an extension created by getenvoy toolkit.
+type ExtensionDir interface {
 	// GetRootDir returns path to the root dir of an extension.
 	GetRootDir() string
```

Comment:
maybe we could delete this interface or merge this to GlobalOpts?

Reply:
maybe but not now, as this whole directory tree is complicated I hope it can disappear, but it was too much to comprehend at the time and logically at least something we can identify as redundant vs before.